collectors/bzimport/convertors.py

```diff
@@ -640,12 +640,20 @@ def get_meta_attr(self, cve_id):
 
     def get_nvd_cvss2(self, cve_id):
         """get NVD CVSS2"""
-        if cve_id in self._nvd_cvss and "cvss2" in self._nvd_cvss[cve_id]:
+        if (
+            cve_id is not None
```

Comment:
Can `_nvd_cvss` contain flaws with `None` as `cve_id`? I feel like adding the `cve_id is not None` is not necessary

Reply:
I think that you are right. I was overly cautious but the CVSS map should only contain CVE keys only - no None key.